docs/source/tensorboard.rst

```diff
@@ -0,0 +1,48 @@
+torch.utils.tensorboard
+===================================
+
+.. note::
+    TensorBoard logging is used to visualize PyTorch models and metrics
+    during their training process. Scalars, images, histograms, graphs, and
+    embedding visualizations are supported for PyTorch models and tensors
+    as well as Caffe2 nets and blobs.
```

Comment:
What I have been able to infer, from reading this PR, is that the way this Tensorboard integration works, is that we write appropriately formatted protobuf records to a directory; then, we point Tensorboard at that directory, it slurps in the data and gives some visualization. To me, just saying "TensorBoard logging" isn't enough specificity for me to infer this. Imagine that I'm a user who wants some visualization, and all I know is that "Tensorboard" is some visualizer thing that I can use.

If the intent is that you're supposed to know what Tensorboard is and how it works, then we should just link to the tensorboard documentation here and have people read that first, if they're not familiar.



If the intent is that a user of the tensorboard

Reply:
To put it on other words, my preference is in this documentation, we essentially replicate the explanation in https://www.tensorflow.org/tensorboard/r1/summaries but with PyTorch clothes. (And I think https://www.tensorflow.org/tensorboard/r1/summaries is the link I was asking for here; put it more up top.)

Reply:
Hopefully the updates are sufficient. Let me know.